assets/base.css

```diff
@@ -3011,3 +3011,17 @@ details-disclosure > details {
     outline: transparent solid 1px;
   }
 }
+
+/* Ambient animation */
+
+@media (prefers-reduced-motion: no-preference) {
+  .animate--ambient > img,
+  .animate--ambient > svg {
+    animation: animateAmbient 30s linear infinite;
```

Comment:
It was at 40s before, had a feeling it was a little more perceivable without too much in your face. I curious how others feel about it?

Reply:
I keep going back and forth. 😄 I'll leave it at this setting for a minute and let folks weigh on whether they think it's too fast or not.

Reply:
Would using the `.animate--ambient` selector on its own be enough here? Why do we specifically need to target `img` and `svg` elements?